docs/jwst/skymatch/description.rst

```diff
@@ -0,0 +1,239 @@
+of a set of input images and computes the signal offsets for each image that will
+minimize -- in a least squares sense -- the residuals across the entire set.
+This comparison is performed directly on the input images without resampling
+them onto a common grid. The overlap regions are computed directly on the sky
+(celestial sphere) for each pair of input images. By default the sky value
+computed for each image is recorded, but not actually subtracted from the
+images. Also note that the meaning of "sky background" depends on the chosen
+sky computation method.
+
+Assumptions
+-----------
+When matching sky background, the code needs to compute bounding polygon
+intersections in world coordinates. The input images, therefore, need to have
+a valid WCS, generated by the :ref:`assign_wcs <assign_wcs_step>` step.
+
+Algorithm
+---------
+The ``skymatch`` step provides several methods for constant sky background
+value computations.
+
+The first method, called "local", essentially is an enhanced version of the
+original sky subtraction method used in older versions of
+`astrodrizzle <https://drizzlepac.readthedocs.io/en/latest/astrodrizzle.html>`_.
+This method simply computes the mean/median/mode/etc. value of the "sky" separately
+in each input image. This method was upgraded to be able to use DQ flags
+to remove bad pixels from being used in computations of the sky statistics.
+
+In addition to the classic "local" method,
+two other methods have been introduced: "global" and
+"match", as well as a combination of the two -- "global+match".
+
+- The "global" method computes a single sky value across *all*
+  input images. The resulting *single sky value* is then considered to be
+  the background in *all input images*. The global value is always based on
+  the image with the lowest sky signal level.
+
+- The "match" algorithm computes a correction value for each image, such
+  that, when applied to each image, the mismatch between *all* pairs of images
+  is minimized, in the least squares sense. For each pair of images, the sky 
+  mismatch is computed *only* in the regions in which the two images overlap
+  on the sky.
+
+  This makes the "match" algorithm particularly useful
+  for equalizing sky values in large mosaics in which one may have
+  only pair-wise intersection of adjacent images without having
+  a common intersection region (on the sky) in all images.
+
+  Note that if the argument "match_down=True", matching will be done to the image
+  with the lowest sky value, and if "match_down=False" it will be done to the
+  image with the highest value.
+
+- The "global+match" algorithm combines the "global" and "match" methods.
+  It uses the "global" algorithm to find a baseline sky value common to all
+  input images and the "match" algorithm to equalize sky values among images.
+  The direction of matching (to the lowest or highest) is again controlled by
+  the "match_down" argument.
```

Comment:
What is the difference between "global+match" and "local"? The example in the table below (very nice) shows that the result is the same.